traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -857,47 +887,46 @@ SELECT
 FROM
   deliveryservice ds
   JOIN type tp ON ds.type = tp.id
-WHERE
-  ds.id = $1
 `
-	di := DSInfo{ID: id}
-	if err := tx.QueryRow(qry, id).Scan(&di.Name, &di.Type, &di.EdgeHeaderRewrite, &di.MidHeaderRewrite, &di.RegexRemap, &di.SigningAlgorithm, &di.CacheURL, &di.MaxOriginConnections, &di.Topology, &di.CDNID); err != nil {
+
+func scanDSInfoRow(row *sql.Row) (DSInfo, bool, error) {
+	di := DSInfo{}
+	if err := row.Scan(
+		&di.ID,
+		&di.Name,
+		&di.Type,
+		&di.EdgeHeaderRewrite,
+		&di.MidHeaderRewrite,
+		&di.RegexRemap,
+		&di.SigningAlgorithm,
+		&di.CacheURL,
+		&di.MaxOriginConnections,
+		&di.Topology,
+		&di.CDNID,
+	); err != nil {
 		if err == sql.ErrNoRows {
 			return DSInfo{}, false, nil
 		}
-		return DSInfo{}, false, fmt.Errorf("querying delivery service server ds info '%v': %v", id, err)
+		return DSInfo{}, false, fmt.Errorf("querying delivery service server ds info: %v", err)
```

Comment:
Why remove the Delivery Service ID from this line? There is nothing else to identify the Delivery Service if this error appears in the logs.

Reply:
I think because if the scan failed, we probably don't have the ID or the xmlID. I will add the ID/XMLID to the callers.